workshops/static/import_from_url.js

```diff
@@ -8,6 +8,7 @@ function import_from_url(url) {
     $("#id_start").val(data.start);
     $("#id_end").val(data.end);
     $("#id_reg_key").val(data.reg_key);
+    $("#id_tags").val(data.tags);
```

Comment:
Without a conditional, `$("#id_tags").val(data.tags);` does not throw any TypeError. If only resets the selection within the `tags` fields.
Are we interested in clearing the selection after an `import` command?